Firestore/core/src/local/leveldb_key.cc

```diff
@@ -567,6 +587,21 @@ std::string Reader::Describe() {
       if (ok_) {
         absl::StrAppend(&description, " query_name=", query_name);
       }
+    } else if (label == ComponentLabel::IndexId) {
+      int32_t index_id = ReadIndexId();
+      if (ok_) {
+        absl::StrAppend(&description, " index_id=", index_id);
+      }
+    } else if (label == ComponentLabel::IndexArrayValue) {
+      const std::string& value = ReadIndexArrayValue();
```

Comment:
Don't store temporary in a const reference.

Reply:
Done.